workshop/content/concepts.adoc

```diff
@@ -1,21 +1,21 @@
 Tekton defines a number of link:https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/[Kubernetes custom resources] as building blocks in order to standardize pipeline concepts and provide a terminology that is consistent across CI/CD solutions. These custom resources are an extension of the Kubernetes API that let users create and interact with these objects using the OpenShift CLI (`oc`), `kubectl`, and other Kubernetes tools.
 
-The custom resources needed to define a `pipeline` are listed below:
+The custom resources needed to define a pipeline are listed below:
 
-* `Task`: a reusable, loosely coupled number of steps that perform a specific task (e.g., building a container image)
-* `Pipeline`: the definition of the `pipeline` and the `tasks` that it should perform
-* `PipelineResource`: inputs (e.g., git repository) and outputs (e.g., image registry) to and out of a `pipeline` or `task`
-* `TaskRun`: the result (i.e., success or failure) of running an instance of `task`
-* `PipelineRun`: the result (i.e., success or failure) of running an instance of `pipeline`, which includes a number of `taskruns`
+* `Task`: a reusable, loosely coupled number of steps that perform a specific task (e.g. building a container image)
+* `Pipeline`: the definition of the pipeline and the tasks that it should perform
+* `PipelineResource`: inputs (e.g. git repository) and outputs (e.g. image registry) to and out of a pipeline or task
+* `TaskRun`: the execution and result (i.e., success or failure) of running an instance of task
```

Comment:
Changed to address the fact that a pipelinerun/taskrun actually execute pipelines/tasks

Reply:
I like the changes. One nit: remove the comma after i.e.  :)

Reply:
Removed the commas